introduction.md

```diff
@@ -1,3 +1,11 @@
 Hi, my name is Chris
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?